src/clientiface.cpp

```diff
@@ -18,6 +18,7 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 */
 
 #include <sstream>
+#include <cmath>
```

Comment:
Seems i was wrong about this, only std::fmin/fmax needs it, so i guess best remove it as is unnecessary?

Reply:
Sure. Let's also test the change.